vendor/diff-parsers-ext-parsewyc.patch

```diff
@@ -178,6 +178,22 @@
  
  let make_loc (startpos, endpos) = {
 @@@@
+   Attr.mk ~loc:Location.none local_ext_loc (PStr [])
+ 
+ let local_extension =
+   Exp.mk ~loc:Location.none (Pexp_extension(local_ext_loc, PStr []))
+ 
+-let include_functor_ext_loc = mknoloc "extension.include_functor"
```

Comment:
I think you should just ignore these patch files.  They are generated files that just document differences from the upstream parser - no semantic effect.  

It's likely someone (me) forgot to commit them with a previous change.  This is fixed in the PR that cleans things up and rebases onto modern ocamlformat, but we haven't been able to merge that yet because of other formatting changes.  It should be fine to just merge whatever got generated here.

Reply:
OK.... but that's harder to review. That is, it sounds like you're suggesting that I examine e.g. parse-wyc/lib/parser.mly. Yet that file includes both the upstream changes and any local changes. The upstream changes, though, aren't really at issue here -- it's just the changes that are the diff between this repo and upstream that I would like to review. I can't easily spot these in the parser.mly file. However, I would imagine that this diff file, though autogenerated, would capture exactly the interesting bit.

It sounds, though, that this diff file in this PR captures aspects that aren't related to this PR, which also makes this file hard to review. So... :shrug:

Reply:
FWIW, I have found it easier to review the diff on the parser than the diff of diffs you are suggesting you want to review (because it typically requires the same context you need to review the parser itself).  But one way to enable the review you want would be to refactor this branch with a commit at the beginning that just includes the changes to the patch files that should have been there, and another commit at the end with all the other patch file changes (at some cost to Antal's sanity during the rebase).

Reply:
I’m perfectly happy to do that, that should be a free rebase

Reply:
I don't feel strongly at all here -- it's just that, as it currently sits, this content is unreviewed, mostly because I don't see a good way to review it. Either I look at this diff file, which contains a bunch of stuff unrelated to this patch; or I look at the change to the parser itself, which also contains a bunch of stuff unrelated to this patch (because it's in the upstream patch). If the actual content is tiny (or non-existent, because there are no actual changes related to this patch? possible.) and you're confident to let it go unreviewed, then that's fine by me.

Reply:
The patch files are now available in 95c808b!

Reply:
That commit looks good. Consider this reviewed. Thanks!